main/src/main/java/org/mobilitydata/gtfsvalidator/validator/StopLatLongValidator.java

```diff
@@ -0,0 +1,55 @@
+package org.mobilitydata.gtfsvalidator.validator;
+
+import org.mobilitydata.gtfsvalidator.annotation.GtfsValidator;
+import org.mobilitydata.gtfsvalidator.notice.NoticeContainer;
+import org.mobilitydata.gtfsvalidator.notice.SeverityLevel;
+import org.mobilitydata.gtfsvalidator.notice.ValidationNotice;
+import org.mobilitydata.gtfsvalidator.table.GtfsLocationType;
+import org.mobilitydata.gtfsvalidator.table.GtfsStop;
+
+/**
+ * Validates {@code stops.stop_lat} and {@code stops.stop_long} exist for a single {@code GtfsStop}
+ * of type Stop, Station, Entrance, and Exit.
+ *
+ * <p>Generated notices:
+ *
+ * <ul>
+ *   <li>{@link LatLongRequiredForStopType}
+ * </ul>
+ */
+@GtfsValidator
+public class StopLatLongValidator extends SingleEntityValidator<GtfsStop> {
```

Comment:
As mentioned offline, I think this class name is a little too close to the name of the existing `GtfsStopLatLonValidator` that it's probably inviting confusion (case in point, I already managed to confuse myself once here already).  I'd like to propose an alternate name.  Thoughts on one of the following?
* `StopRequiredLocationValidator`
* `StopConditionallyRequiredLocationValidator`

Basically, capturing that this validator is checking the presence of the location data itself, as opposed to validating the values in some way.  Thoughts?

Reply:
@bdferris-v2 Sure - I'm happy to change it.  No strong preference unless we want to get very specific (and long) with something like `LocationRequiredForStopStationEntranceValidator` or `StopStationEntranceLocationRequiredValidator` 😬Whatever feels more in line with the current naming convention...

Reply:
I'm not a Java long naming absolutist :)  I'd just go with `StopRequiredLocationValidator`